lib/geo/database.ex

```diff
@@ -4,11 +4,11 @@ defmodule GEO.Database do
   @doc """
   Returns the path to the GeoIP database tarball.
   """
-  @spec source_path(file_name :: binary) :: binary
-  def source_path(file_name \\ "ip_database.tar.gz") do
-    Path.join(
-      Application.get_env(:geo, :database_path, "#{:code.priv_dir(:geo)}"),
-      file_name
+  @spec source_path() :: binary
+  def source_path() do
+    System.get_env(
+      "GEOLIX_DATABASE_PATH",
+      Path.expand("#{__DIR__}/../../../priv/geo/ip_database.tar.gz")
```

Comment:
Since we already have this configured in `config`, and we only have one database, maybe it should read the application config directly, instead of reading the env variable values twice? It's also inconsistent with handling default values: why do we have `get_env` here, but in `runtime.exs` we use `fetch_env`?

Reply:
it is a bit annoying to do, we should still improve that.